frontend/admin/src/js/components/user/UserTable.tsx

```diff
@@ -324,15 +324,39 @@ export const UserTable = ({ initialFilterInput }: UserTableProps) => {
       },
       {
         label: intl.formatMessage({
-          defaultMessage: "Preferred Language",
-          id: "mf+QEY",
+          defaultMessage: "Preferred Communication Language",
+          id: "CfXIqC",
           description:
-            "Title displayed for the User table Preferred Language column.",
+            "Title displayed for the User table Preferred Communication Language column.",
         }),
         accessor: (user) => languageAccessor(user.preferredLang, intl),
         id: "preferredLanguage",
         sortColumnName: "preferred_lang",
       },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Preferred Spoken Interview Language",
+          id: "iRJV64",
+          description:
+            "Title displayed on the Pool Candidates table Preferred Spoken Language column.",
+        }),
+        id: "preferredLang",
+        accessor: (user) =>
+          languageAccessor(user?.preferredLanguageForInterview, intl),
+        sortColumnName: "PREFERRED_LANGUAGE_FOR_INTERVIEW",
```

Comment:
I'm not entirely sure how sortColumnName works, but is there any reason its in all-caps here and not on other columns?

Reply:
no .. thats by mistake ... good catch!